src/xdocs/config_misc.xml

```diff
@@ -28,8 +28,16 @@
         <p>
           Checks the style of array type definitions.  Some like Java style:
           <code>public static void main(String[] args)</code> and some like
-          C style: public static void main(String args[])
+          C style: <code>public static void main(String args[])</code>.
         </p>
+        <p>
+          This check also enforces the style of method return types:
+        </p>
+        <source>
+public byte getData()[] // violation (for both Java and C style)
```

Comment:
Java code with violation and ok messages should be in `Examples` section.
You can view some of our `naming` checks to see how we are re-building our examples.